llm-miner-starter.sh

```diff
@@ -285,12 +285,18 @@ validateVram() {
     fi
 
     # Determine GPU memory utilization based on model name and available VRAM
-    if [[ "$heurist_model_id" == *"mixtral-8x7b-gptq"* ]] && [ "$total_available_mb" -gt 35000 ]; then
-        local gpu_memory_util=$(echo "scale=2; (35000-1000)/$total_available_mb" | bc)
-    elif [[ "$heurist_model_id" == *"70b"* ]] && [ "$total_available_mb" -gt 44000 ]; then
-        local gpu_memory_util=$(echo "scale=2; (44000-1000)/$total_available_mb" | bc)
-    elif [[ "$heurist_model_id" == *"8b"* ]] && [ "$total_available_mb" -gt 21000 ]; then
-        local gpu_memory_util=$(echo "scale=2; (21000-1000)/$total_available_mb" | bc)
+    if [[ "$heurist_model_id" == *"mixtral-8x7b-gptq"* ]] && [ "$available_mb" -gt 32000 ]; then
```

Comment:
let's use the original version for this section?
I increased vram usage after setting context length to 8k (https://github.com/heurist-network/miner-release/commit/08bfd20d815ebae9163db276b6d0a82faca4c93f)

Reply:
reset to the original version